data/s390x/TOOL_s390_ZIOMON/ziomon_basic.pl

```diff
@@ -81,7 +81,7 @@ sub reset_and_show_dmesg
     open my $fh, '>', "temp" or die $!;
     print $fh `dmesg -c`;
     close($fh);
-    &filter_repeated("temp", "<DMESG>|");
+    filter_repeated("temp", "<DMESG>|");
```

Comment:
please exclude data/ - it's not our code

Reply:
Now I understand, for perl is not our code but for other files (like yast profiles) it is. Good point. Thanks. Changed, not file changed in folder data/ in any commit.